webpack.config.js

```diff
@@ -0,0 +1,35 @@
+var webpack = require('webpack');
+
+module.exports = {
+  entry: './src/index',
+  module: {
+    loaders: [
+      {
+        test: /\.js$/,
+        loader: 'babel',
+        exclude: /node_modules/,
+        query: {
```

Comment:
@dhlavaty babel-loader uses `.babelrc` when available – it should be safe to remove the `query` object here, so we don't need to update two files when the babel config changes.